ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -70,9 +79,11 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
 
             final OFMeetConfig ofMeetConfig = new OFMeetConfig();
 
+			final boolean isSwitchAvailable = JiveGlobals.getBooleanProperty("freeswitch.enabled", false);
             final String xmppDomain = XMPPServer.getInstance().getServerInfo().getXMPPDomain();
+            final String sipDomain = JiveGlobals.getProperty("freeswitch.sip.hostname", getIpAddress());
 
-            final JSONArray conferences = new JSONArray();
+            final JSONObject conferences = new JSONObject();
```

Comment:
On the client in ofmeet.js. It is used to pass on the list of conferences to the SIP integration.